client/src/components/Navbar/Navbar.js

```diff
@@ -1,21 +1,102 @@
-import React from 'react';
+import React from "react";
 import "./Navbar.css";
-import Logo from "../Logo/Logo"
+import Logo from "../Logo/Logo";
+import Burger from "../Burger/Burger";
+import ProfilePicture from "../ProfilePicture/ProfilePicture";
+import NavLink from "../NavLink/NavLink";
+import Icon from "../Icon/Icon";
+import PropTypes from "prop-types";
 
 // Function based React Component
-const Navbar = (props) => {
-
+const Navbar = props => {
   // Default Class to apply to Component
-  let classList = `Navbar`;
-
 
   return (
-    <div className={classList}>
-      <div className="navbar-logo">
+    <div className="Navbar">
+      <div className="Navbar__logo">
         <Logo clickable to="/route" />
       </div>
+      <div className="Navbar__main">
+        <div className="Navbar__menu">
+          <div className="Navbar__link-item">
+            <NavLink
+              text="My Directory"
+              to="/"
+              menuList={[
+                { text: "Link 1", to: "/" },
+                { text: "Link 2", to: "/" }
+              ]}
+            />
+            <div className={`Navbar__triangle ${props.menu1}`}></div>
+          </div>
+          <div className="Navbar__link-item">
+            <NavLink
+              text="Community"
+              to="/"
+              menuList={[
+                { text: "Link 1", to: "/" },
+                { text: "Link 2", to: "/" }
+              ]}
+            />
+            <div className={`Navbar__triangle ${props.menu2}`}></div>
+          </div>
+          <div className="Navbar__link-item">
+            <NavLink
+              text="Forum"
+              to="/"
+              menuList={[
+                { text: "Link 1", to: "/" },
+                { text: "Link 2", to: "/" }
+              ]}
+            />
+            <div className={`Navbar__triangle ${props.menu3}`}></div>
+          </div>
+          <div className="Navbar__link-item">
+            <NavLink
+              text="Book A Trip"
+              to="/"
+              menuList={[
+                { text: "Link 1", to: "/" },
+                { text: "Link 2", to: "/" }
+              ]}
+            />
+            <div className={`Navbar__triangle ${props.menu4}`}></div>
+          </div>
```

Comment:
Can you make this a list?

Reply:
I actually don't see where you're using these props in the component. Could they be removed entirely?

Reply:
They were just there to show that the dropdown menu works. Nothing drops down if that array is empty. So, they can be removed.